vm/inc/ubpf.h

```diff
@@ -63,6 +65,22 @@ extern "C"
      */
     typedef uint64_t (*ubpf_jit_fn)(void* mem, size_t mem_len);
 
+    /**
+     * @brief Opaque type for a uBPF JIT compiled function with
+     *        external stack.
+     */
+    typedef uint64_t (*ubpf_jit_ex_fn)(void* mem, size_t mem_len, uint8_t* stack, size_t stack_len);
+
+    /**
+     * @brief Enum to describe JIT mode.
```

Comment:
Can we please document what these two options mean?

Reply:
YES! (I am not sure how I omitted this documentation -- sorry!)